### DISCUSSION TITLE
[Bug] message_from_seller is nil

### DISCUSSION THREAD

**Discussion by joshrendek** [etsy/open-api#890] [Q&A]
The v3 API response for receipts on both the list and single receipt endpoints don't return the private note set by the buyer, they just return null/nil. 

I can see the private note in the Etsy UI, but when retrieved via the API I only get the buyers not, not my seller note:

```
 "message_from_payment"=>nil,
 "message_from_seller"=>nil,
 "message_from_buyer"=>"<buyers msg here>",
```

**Comment by turboturtleD** [comment]
Last I checked, only the buyer note is available on the API.  This is the same as in v2, so unfortunately not a bug.  While it isn’t convenient, we just make the seller notes within our app (and don’t use the Etsy seller notes data field), then attach it to receipt ids in our app db.

**Reply by joshrendek** [reply]
Nothing in the docs indicate this though - so it is a bug https://developer.etsy.com/documentation/reference/#operation/getShopReceipt

**Reply by turboturtleD** [reply]
The message_from_seller is not the same thing as the seller private notes.  message_from_seller is a message sent to the buyer when they purchase based on a setting that can be located in the front end GUI under "Settings > Info & Appearance > Message to Buyers".  Not sure if there is access in the API for this setting, but it is not the same as seller private notes as this message is much more global.

**Reply by turboturtleD** [reply]
It looks like this can be changed in the updateShop endpoint under sale_message (or digital_sale_message)
